src/fal/planner/plan.py

```diff
@@ -41,16 +67,73 @@ def from_execution_plan(
 
 
 @dataclass
-class PlannedGraph(OriginGraph):
+class ShuffledGraph(OriginGraph):
     graph: nx.DiGraph
+    after_pattern = re.compile("^script\\..+\\.AFTER\\..+\\.(py|ipynb)")
+    before_pattern = re.compile("^script\\..+\\.BEFORE\\..+\\.(py|ipynb)")
```

Comment:
ATL: why not use `fal.cli.selectors._is_before_script`?
https://github.com/fal-ai/fal/blob/4373278c33c8b34214a434d127cb82d2de1b3a96/src/fal/cli/selectors.py#L287-L296

Reply:
I had forgotten about this. I'll use it